code-challenges.test.js

```diff
@@ -70,4 +126,26 @@ const accountTransactions3 = []
 
 // b) Create the function that makes the test pass.
 
+//Jest Testing
+// describe("accumulatingSum", () => {
+//   it("Returns the accumulating sum of the array. If an empty array, then it should return an empty array.", () => {
+//     expect(accumulatingSum(accountTransactions1)).toEqual([100, 83, 60, 51])
+//     expect(accumulatingSum(accountTransactions2)).toEqual([250, 161, 261, 165])
+//     expect(accumulatingSum(accountTransactions3)).toEqual([])
+//   })
+// })
+
 // Pseudo code:
+//Function Name: accumulatingSum
+//Input: Array of numbers
+//Output: Array of accumulated sum of the numbers inside.
+//Process: .map() to create a new array with the given information. 
+
+const accumulatingSum = (arr) => {
+    let addSum = 0
+    return arr.map((value) => addSum += value)
+    //Not understanding how addSum += value works. Could you explain it. Please and thank you
```

Comment:
The function starts by declaring a variable `addSum` and starting it at 0. Then we are iterating through the array using .map, which will return something for each value in the array. The logic in the .map is stating that you want to add the current value in the array to the addSum variable, then return the sum as the value in the new array.